lib/ansible/plugins/doc_fragments/action_common_attributes.py

```diff
@@ -14,6 +14,13 @@ class ModuleDocFragment(object):
     action:
       description: Indicates this has a corresponding action plugin so some parts of the options can be executed on the controller
       support: none
+    action_group:
+      description: If module_defaults are settable for this action, as it is part of an action_group
```

Comment:
The wording is confusion since any module, regardless of inclusion in a group can use
```
module_defaults:
  ns.coll.module:
    ...
```

Reply:
'is already included' ?

Reply:
that is what the membership list is for, just need to indicate this is already in a group (or requesting it in future to avoid hardcoded lists)

Reply:
Oh, I see. 'If module_defaults are settable are settable for this action by way of an action_group'?

Reply:
yeah, will try to make it clearer, had hard time phrasing this one.

Reply:
Looks good to me